src/resources/js/event-editor.js

```diff
@@ -12,101 +12,10 @@ var tribe_events_event_editor = tribe_events_event_editor || {};
 	 * @since 6.0.1
 	 */
 	obj.selectors = {
-		addButton: '.tribe-add-post',
-		deleteButton: '.tribe-delete-this',
-		dropDown: '.linked-post-dropdown',
 		featuredEventCheckbox: 'input[name="feature_event"]',
-		finalDropDown: '.linked-post-dropdown:last-of-type',
-		linkedPostWrapper: '.linked-post-wrapper',
-		savedItems: '.saved-linked-post',
 		stickyInMonthViewCheckbox: 'input[name="EventShowInCalendar"]',
 	};
 
-	obj.linkedPost = {};
-
-	/**
-	 * Controls logic for the Organizer delete button to display.
-	 *
-	 * If more than one organizer exists, display the delete button.
-	 * If only one organizer exists, hide the delete button.
-	 *
-	 * @since 6.0.1
-	 * @return void
-	 */
-	obj.linkedPost.deleteButtonDisplayLogic = function () {
```

Comment:
This is all of the logic for hiding and displaying the trash and add buttons, which is still needed. 

When there are no venues or organizers on the post, the add and trash buttons should not be visible. 

When there is a filled in venue or organizer, the button should be visible.

Venues should only have and add venue button and trash button when multi venues are enabled.

Reply:
Resolved these with the latest set of changes..